contrib/python-bindings/source/CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ INCLUDE(FindPythonLibs)
 INCLUDE(FindPythonInterp)
 
 #
+# Unset Boost_Found and run the low level FindBOOST.cmake module again to
+# pick up libboost_python.so
+#
+LIST(REMOVE_ITEM CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules/)
+SET(Boost_FOUND)
+# Use the low level _FIND_PACKAGE function instead of our wrapper
+_FIND_PACKAGE(Boost 1.54 COMPONENTS python)
```

Comment:
should we set `DEAL_II_COMPONENT_PYTHON_BINDINGS` to `OFF` if boost+python is not found? I don't think it's done anywhere. Of throw an error?